joplin/js/editor.js

```diff
@@ -88,7 +88,7 @@ $(function() {
       $(label).append(link);
     }
   }
-
+  // TODO: what is this for??
```

Comment:
Looks like it came from this: https://github.com/cityofaustin/joplin/commit/a067fb95ae0b00bf1e718503b02d068e02aa7a0d